src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,117 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
 
+import cn from 'classnames';
+import PropTypes from 'prop-types';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+import { getMovies } from '../../api/api';
+
+export const FindMovie = ({ addMovie, addedMovies }) => {
+  const [search, setSearch] = useState('');
+  const [movie, setMovie] = useState({});
+  const [errorMessage, seterrorMessage] = useState('');
+
+  const searchMovie = () => {
+    getMovies(search)
+      .then((foundMovie) => {
+        if (foundMovie.Response === 'False') {
+          seterrorMessage('Can\'t find a movie with such a title');
+
+          return;
+        }
+
+        const newMovie = {
+          title: foundMovie.Title,
+          description: foundMovie.Plot,
+          imgUrl: foundMovie.Poster,
+          imdbUrl: `https://www.imdb.com/title/${foundMovie.imdbID}`,
+        };
+
+        setMovie(newMovie);
+        seterrorMessage('');
+      });
+  };
+
+  const addMovieToList = () => {
+    if (addedMovies
+      .filter(addedMovie => addedMovie.imdbId === movie.imdbID).length === 0) {
```

Comment:
```suggestion
    addedMovies.find(addedMovie => addedMovie.imdbId === movie.imdbID)
```
краще не ходити обхідними шляхами, а просто знайти цей елемент, і зберегти в змінній, потім в if просто перевіряти чи він є